packages/ffmpeg.rb

```diff
@@ -218,5 +220,8 @@ def self.build
   def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
     FileUtils.install 'tools/qt-faststart', "#{CREW_DEST_PREFIX}/bin/", mode: 0o755
+    FileUtils.mkdir_p "#{CREW_DEST_MAN_PREFIX}/man1/"
```

Comment:
Not needed with new FileUtils.install update?

Reply:
Fixed.